tests/sles4sap/patterns.pm

```diff
@@ -31,6 +31,13 @@ sub run {
 
     my $base_pattern = is_sle('>=15') ? 'patterns-server-enterprise-sap_server' : 'patterns-sles-sap_server';
 
+    # In case of DVD based installation we need to (re-)enable the local repos
+    # Non-SCC installation usually means installation from DVD
+    assert_script_run 'zypper mr -e -l'
+      if is_sle('15+')
+      and !check_var('SCC_REGISTER', 'installation')
```

Comment:
I wonder what would happen in this condition when using `sles4sap/patterns` after an offline migration. I recommend including to the condition a check based on `UPGRADE` or `FULL_UPDATE` to cover that.

Reply:
Yes.

Current test are not really offline as they are using FTP, I will try to do a real offline test to check.